src/utilities/getContractAddress.ts

```diff
@@ -1,8 +1,13 @@
+import isolatedLendingTestnetDeployments from '@venusprotocol/isolated-pools/deployments/bsctestnet.json';
 import config from 'config';
 
 import mainContractAddresses from 'constants/contracts/addresses/main.json';
 
 const getContractAddress = (contractId: keyof typeof mainContractAddresses) =>
   mainContractAddresses[contractId][config.chainId];
 
+export const getIsolatedPoolAddress = (
+  contractName: keyof typeof isolatedLendingTestnetDeployments.contracts,
+) => (config.isOnTestnet ? isolatedLendingTestnetDeployments.contracts[contractName].address : '');
```

Comment:
Rather than using a separate util, how about keeping the existing logic and making a list of all the contracts (the ones we keep a reference of locally merged with the ones taken from the package)?

Something like:
```typescript
const isolatedLendingContractAddresses = Object.entries(
  isolatedLendingTestnetDeployments.contracts,
).map(([contractName, contractInfo]) => ({
  [contractName]: {
    97: contractInfo.address,
    56: '', // TODO: fill up once contracts have been deployed to mainnet
  },
}));

const contractAddress = {
  ...mainContractAddresses,
  ...isolatedLendingContractAddresses
};
```

This way `getContractAddress` can stay the same and the logic for retrieving contract addresses within the dApp can remain as it is.